### PR TITLE
Added content installation to instance test too

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [x] Ready

## Description
Added content installation to instance test too.

### CODE REVIEW

File: Tests/instance_notifier.py [hunk 1]
```diff
@@ -43,6 +48,7 @@ def test_instances(secret_conf_path, server, username, password):
 
     for integration in integrations:
         c = demisto_client.configure(base_url=server, username=username, password=password, verify_ssl=False)
+        install_new_content(c, server)
```
Comment (yaakovi) [inline Tests/instance_notifier.py]: why is that inside the loop?

File: Tests/instance_notifier.py [hunk 2]
```diff
@@ -24,6 +24,11 @@ def options_handler():
     return options
 
 
+def install_new_content(client, server):
+    prints_manager = ParallelPrintsManager(0)
+    update_content_on_demisto_instance(client, server, prints_manager, 0)
```
Comment (yaakovi) [inline Tests/instance_notifier.py]: print the collected messages after calling the function:
`prints_manager.execute_thread_prints(0)`

File: Tests/instance_notifier.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Tests/instance_notifier.py]: True

Comment (ShahafBenYakir) [reply Tests/instance_notifier.py]: True

File: Tests/instance_notifier.py (hunk 2 above)
Comment (ShahafBenYakir) [reply Tests/instance_notifier.py]: True

File: Tests/instance_notifier.py [hunk 3]
```diff
@@ -27,6 +27,7 @@ def options_handler():
 def install_new_content(client, server):
     prints_manager = ParallelPrintsManager(0)
```
Comment (yaakovi) [inline Tests/instance_notifier.py]: missed that the first time
```suggestion
    prints_manager = ParallelPrintsManager(1)
```

**Comment by yaakovi** [comment]
forced pushed, failed on flashpoint test (unrelated to changes in the PR)

**Pull request merged**